cli/azd/pkg/input/console.go

```diff
@@ -86,6 +86,8 @@ type Console interface {
 	Confirm(ctx context.Context, options ConsoleOptions) (bool, error)
 	// block terminal until the next enter
 	WaitForEnter()
+	// Writes a new line to the writer if there if the last character written was not '\n'
```

Comment:
The implementation takes into account the last two bytes written, not just the last byte, as the comment here would lead you to believe, correct?

Perhaps this should be something like `EnsureBlankLine()`?  That's what we are really after here, right?